pyproject.toml

```diff
@@ -27,7 +27,7 @@ dependencies = [
   "conda",
   "keyring",
   "requests",
-  "ruamel.yaml<0.18",
```

Comment:
This was neeeded since otherwise the `pip install -e .` call would downgrade the dependency